src/main/java/io/lettuce/core/AbstractRedisClient.java

```diff
@@ -297,6 +301,17 @@ protected void channelType(ConnectionBuilder connectionBuilder, ConnectionPoint
         }
```

Comment:
[Supplemental Comments]
I searched `new Bootstrap()` in source codes and `AbstractRedisClient#connectionBuilder` is the only place where `Bootstrap` is used.
I confirmed all return values of `connectionBuilder()` is passed to `resolver(...)`